script.logviewer/addon.xml

```diff
@@ -21,6 +21,8 @@
         <news>
 - Python 3 compatibility
 - Fix log error detection service after kodi's migration to libfmt
+- Fix android file handling
```

Comment:
News should be only for the newest release. You're including the changes from the previous version too